src/components/EventWarning/index.js

```diff
@@ -4,47 +4,31 @@ import { withStyles } from 'material-ui/styles';
 import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
 import classNames from 'classnames';
 
-import { EventWarningType } from '../../constants';
 import styles from './styles';
 
-class EventWarning extends React.PureComponent {
-  render() {
-    const {
-      classes,
-      className,
-      typeClass,
-      message,
-      intl,
-      theme,
-      ...props
-    } = this.props;
 
-    if (!message) {
-      return null;
-    }
+const FormattedMessageFixed = (props) => <FormattedMessage {...props} />; // workaround: http://bit.ly/2u7Hhe4
```

Comment:
why is this necessary?

Reply:
did you reqad the url in the comments? :)

Reply:
somehow the url doesn't work for me